src/GafferOSL/ShadingEngine.cpp

```diff
@@ -912,7 +927,62 @@ OSL::ShadingSystem *shadingSystem()
 
 	g_shadingSystem->attribute( "commonspace", "object" );
 
-	return g_shadingSystem;
+	bool requestBatch = true;
+	if( const char *requestBatchVar = getenv( "GAFFER_OSL_USESIMD" ) )
```

Comment:
GAFFEROSL_USE_SIMD, to match the convention used by our other env vars.

Reply:
Addressed in ae15e51